doc/subsystems/logging/logger.rst

```diff
@@ -0,0 +1,255 @@
+.. _logger:
+
+Logger
+######
+
+The log API provides a common interface to process messages issued by developers. These messages are processed by active backends. Currently only UART backend is available. However, generic backend interface exists and more backends can be added and easily plugged in to the system.
```

Comment:
Even thou the generated documentation will not care about long lines, it would be better if you try to keep the lines max 80 char long. This way it is easier for a human to read the .rst file.

Reply:
changed to fit all lines in 80 chars.